terraform/environments/staging/variables.tf

```diff
@@ -27,7 +32,26 @@ variable "session_bucket_ttl_days" {
   default     = 14
 }
 
-variable "ops_project_id" {
+variable "setup_cd_system" {
+  type        = bool
+  default     = false
+  description = "Create deployment triggers. Enable only if Cloud Build has GitHub repository access."
+}
+
+variable "repo_owner" {
+  description = "Repo Owner (GoogleCloudPlatform)"
+  type        = string
+  default     = ""
```

Comment:
Should we specify `GoogleCloudPlatform` as a default here?

Reply:
I left this unchanged from earlier implementation. We probably shouldn't set this as default, because we're the only team that should be wiring up Cloud Build to the canonical repository.